src/providers/built-in-providers.ts

```diff
@@ -212,7 +212,12 @@ export const ollamaProvider: IProviderInfo = {
   apiKeyRequirement: 'none',
   defaultModels: [],
   supportsBaseURL: true,
-  supportsHeaders: true,
+  URLS: [
```

Comment:
Wondering if this should be named `baseUrls` to be consistent with the user facing string?

Reply:
Right, I updated it.